pages/apis/api_differences.md.erb

```diff
@@ -0,0 +1,48 @@
+# API Differences Between REST and GraphQL
+
+Buildkite provides both a [REST API](https://buildkite.com/docs/apis/rest-api) and [GraphQL API](/docs/apis/graphql-api), but there is no full parity between the two. Certain actions can only be performed using the REST API, but not GraphQL, and vice versa.
+
+{:toc}
+
+The strengths of the GraphQL API are in complex data queries, and the strengths of the REST API are in creating and modifying records.
+
+On this page we've collected the known limitation where some API features are only available with either REST or GraphQL.
+
+## Features only available in the REST API
```

Comment:
The REST API also exclusively allows showing some information about the access token currently in use, and revoking the current access token:

https://buildkite.com/docs/apis/rest-api/access-token#get-the-current-token